action/action-dsl/src/test/java/com/powsybl/action/dsl/ActionDslLoaderTest.java

```diff
@@ -151,6 +186,70 @@ private void addPhaseShifter() {
                     .setAlpha(5.0)
                     .setRho(6.0)
                 .endStep()
+                .beginStep()
+                    .setR(1.0)
+                    .setX(2.0)
+                    .setG(3.0)
+                    .setB(4.0)
+                    .setAlpha(5.0)
+                    .setRho(6.0)
+                .endStep()
+                .beginStep()
+                    .setR(1.0)
+                    .setX(2.0)
+                    .setG(3.0)
+                    .setB(4.0)
+                    .setAlpha(5.0)
+                    .setRho(6.0)
+                .endStep()
                 .add();
     }
+
+    private static final class DeltaTapData {
```

Comment:
Not very important for such a test class, but setters don't seem useful:
we can make this class immutable (remove setters and have final fields).